ContestUtil/src/org/icpc/tools/contest/util/ImagesGenerator.java

```diff
@@ -434,8 +437,12 @@ private static File getDefaultFile(File folder, String property) {
 	}
 
 	public void generateOrganizationLogos() {
-		ImageSpec[] spec = new ImageSpec[] { new ImageSpec(DESKTOP, false), new ImageSpec(ICON, true),
-				new ImageSpec(TILE, true) };
+		ImageSpec[] spec = new ImageSpec[] { new ImageSpec(DESKTOP, false), new ImageSpec(TILE, true) };
+		int originalLength = spec.length;
+		spec = Arrays.copyOf(spec, spec.length + ICONS.length);
+		for (int i = 0; i < ICONS.length; i++) {
+			spec[originalLength + i] = new ImageSpec(ICONS[i], true);
```

Comment:
This will give you 64x64 padded icons and normally-scaled icons (maybe 64x50 or 57x64), but I thought you said you didn't want that. Or that's fine, you can deal with both/will ignore one?

Reply:
In DOMjudge we prefer the 64x64 padded versions and ignore the rest if it exists

Reply:
There is no option to NOT output the normally-scaled ones right? We indeed need the padded ones.